.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -6,6 +6,7 @@ set -e
 CHAT_ID=$1
 TELEGRAM_BOT_TOKEN=$2
 QUIZAPI_KEY=$3
+skipped_quizzes=0
```

Comment:
```suggestion
SKIPPED_QUIZZES=0
```

Reply:
By convention
Variable Names: Lower-case, with underscores to separate words. Ex: my_variable_name
Constants and Environment Variable Names: All caps, separated with underscores, declared at the top of the file. Ex: MY_CONSTANT

In this case, skipped_quizzes is a variable (the number of quiz skipped)